.golangci.yml

```diff
@@ -18,8 +18,8 @@ linters:
     - unparam
     - unused
     - durationcheck
-    - exportloopref
     - makezero
     - nilerr
     - reassign
-  
\ No newline at end of file
+  disable:
+    - exportloopref
```

Comment:
Why are we disabling this here? Any specific reason?

Reply:
It was making the ci fail with:
`level=error msg="[linters_context] exportloopref: This linter is fully inactivated: it will not produce any reports."`
I'll check if this is related to golangci linter 🤔

Reply:
It is probably related to that. Newer versions of golangci linter deprecates some configs. You can check it, and replace/delete what is needed. But disabling it seems not right. Let's remove it in any case, and next time, maybe add a comment for that.

Reply:
Yesss for sure, thank you Emma!